pages/_app.tsx

```diff
@@ -38,7 +39,7 @@ Layout.propTypes = {
   children: node.isRequired,
 };
 
-function Layout({ children }) {
+function Layout({ children }: PropsWithChildren<{}>) {
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/issues/2063#issuecomment-675156492

```suggestion
function Layout({ children }: PropsWithChildren<unknown>) {
```